core/store/orm/schema.go

```diff
@@ -23,6 +23,8 @@ type ConfigSchema struct {
 	ClientNodeURL                         string          `env:"CLIENT_NODE_URL" default:"http://localhost:6688"`
 	DatabaseTimeout                       models.Duration `env:"DATABASE_TIMEOUT" default:"500ms"`
 	DatabaseURL                           string          `env:"DATABASE_URL"`
+	DatabaseListenerMinReconnectInterval  time.Duration   `env:"DATABASE_LISTENER_MIN_RECONNECT_INTERVAL" default:"1m"`
```

Comment:
`1m` naively seems high. What is the risk with trying to reconnect every 10 seconds?

Reply:
These were just the defaults from the `pq` lib's examples.  Happy to change it but curious to hear @samsondav's take on performance before I do

Reply:
I don't have an opinion on this. Either 1m or 10s is fine by me.